packages/babel-plugin/src/__tests__/module-traversal.test.tsx

```diff
@@ -267,4 +267,33 @@ describe('module traversal', () => {
       );
     }).toThrowErrorMatchingSnapshot();
   });
+
+  it('should inline css from a call expression mixin referencing an identifier from another module', () => {
+    const result = transform(
+      `
+      import '@compiled/core';
+      import { colorMixin } from './stubs/objects';
+
+      <div css={{ ':hover': colorMixin() }} />
+    `
+    );
+
+    expect(result).toInclude('.cc-hash-test:hover{color:red;background-color:pink}');
+  });
+
+  it('should inline css from a member expression mixin referencing an identifier from another module', () => {
```

Comment:
should this be inlining? its resolved to a dynamic value

Reply:
Ohh right. Will update the test statement.

Reply:
could it inline? whats stopping us from doing it? the test above makes it look like it should work

Reply:
So, I am only checking `CallExpression` for identifiers which are inlined in css prop or template literal as of now. I removed that logic of checking `CallExpression` from `getInterpolation` coz it was going beyond the scope of this PR. Do we have card for this behaviour or should I add one 🤔 ?

Reply:
pls add one :)

Reply:
Added https://github.com/atlassian-labs/compiled/issues/285